# DP: Make vhdl known to the PPC backend. See 
# DP: https://github.com/ghdl/ghdl/blob/master/doc/development/building/GCC.rst

--- a/gcc/config/rs6000/rs6000-logue.cc
+++ b/gcc/config/rs6000/rs6000-logue.cc
@@ -5351,7 +5351,8 @@
 	  || ! strcmp (language_string, "GNU Go")
 	  || ! strcmp (language_string, "GNU D")
 	  || ! strcmp (language_string, "GNU Rust")
-	  || ! strcmp (language_string, "libgccjit"))
+	  || ! strcmp (language_string, "libgccjit")
+	  || ! strcmp (language_string, "vhdl"))
 	i = 0;
       else if (! strcmp (language_string, "GNU F77")
 	       || lang_GNU_Fortran ())
